test/core/Safe.Execution.spec.ts

```diff
@@ -328,8 +358,14 @@ describe("Safe", () => {
             await user1.sendTransaction({ to: safeAddress, value: ethers.parseEther("1") });
             await expect(await hre.ethers.provider.getBalance(safeAddress)).to.eq(ethers.parseEther("1"));
 
-            const executedTx = await executeTx(safe, tx, [await safeApproveHash(user1, safe, tx, true)], { gasLimit: 500000 });
+            // await expect(await executeTx(safe, tx, [await safeApproveHash(user1, safe, tx, true)], { gasLimit: 5500000 })).to.emit(
+            //     nativeTokenReceiver,
+            //     "BreadReceived",
+            // );
```

Comment:
nit: left-in commented out code.

Reply:
I left it on purpose because I was looking for a way to make it work in ZkSync, this test should be considered unfinished